man/avahi.service.5.xml.in

```diff
@@ -96,7 +96,37 @@
       </option>
 
       <option>
-        <p><opt>&lt;txt-record&gt;</opt> DNS-SD TXT record data.</p>
+        <p><opt>&lt;name replace-wildcards="yes|no"&gt;</opt> The
```

Comment:
Hm, I remember copying a paragraph as a help to extend the one about TXT records. Probably forgot to remove it later.